src/color-scale/color-scale.js

```diff
@@ -149,6 +416,53 @@ const Self = class ColorScale extends ColorElement {
 			},
 			additionalDependencies: ["info"],
 		},
+		editable: {
+			parse (value) {
+				if (value === undefined || value === null || value === false || value === "false") {
+					return;
+				}
+
+				if (value === "" || value === "editable" || value === true || value === "true") {
+					// Boolean attribute
+					return {
+						name: true,
+						color: true,
+						list: true,
+					};
+				}
+
+				if (typeof value === "string") {
+					// Convert to object
+					let entries = value.split(/\s*[,\s]\s*/).map(key => [key.trim(), true]);
+					return Object.fromEntries(entries);
+				}
+
+				if (typeof value === "object") {
+					return value;
+				}
+
+				console.warn(`The specified value "${ value }" cannot be used as a value of the "editable" property.`);
+				return;
+			},
+			convert (value) {
+				if (value?.list) {
+					// Enable the list operations: add, delete, reorder, etc.
+					delete value.list;
+					return {...value, add: true, delete: true, reorder: true};
+				}
+
+				return value;
+			},
+			reflect: {
+				from: true,
+			},
+		},
+		defaultColor: {
```

Comment:
I think this should also accept a single color as a string or `Color` object (via an attribute too). Don't let the complex case get in the way of the simple one :)

Reply:
Done!